source/_integrations/nanoleaf.markdown

```diff
@@ -22,4 +22,8 @@ The Nanoleaf integration allows you to control and monitor Nanoleaf Light Panels
 
 This integration does not support the Nanoleaf Remote and Essentials lights.
 
+# Transition and brightness
+When using transition in service calls (such as light.turn_on), transition is only applied to brightness and does not apply to color. When a service call has a transition set but no brightness is included in the call, the light will automatically transition to 100% brightness.
```

Comment:
```suggestion
# Transition and brightness

When using a transition in service calls (such as `light.turn_on`), the transition is only applied to brightness and does not apply to color. When a service call has a transition set but no brightness is included, the light will automatically transition to 100% brightness.
```

Reply:
Additionally, let move this section below the config flow section; as this is, of course, only a concern if one has set it up 😉